program-libs/batched-merkle-tree/README.md

```
@@ -0,0 +1,174 @@
+<!-- cargo-rdme start -->
+
+## light-batched-merkle-tree
```

markdownlint-cli2 (0.20.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)